test/ica.jl

```diff
@@ -58,11 +59,12 @@ W = M.W
 @test transform(M, X) ≈ W' * (X .- mv)
 @test W'W ≈ eye(k)
 
-M = fit(ICA, X, k; do_whiten=true)
+M = fit(ICA, X, k; do_whiten=true, tol=Inf)
 @test isa(M, ICA)
 @test indim(M) == m
 @test outdim(M) == k
 @test mean(M) == mv
 W = M.W
 @test W'C * W ≈ eye(k)
 
+@test_throws StatsBase.ConvergenceException fit(ICA, X, k; do_whiten=true, tol=1)
```

Comment:
With the most recent changes, the call to `fit` does not fail to converge and so the test fails.